src/ocspdash/web/blueprints/api.py

```diff
@@ -65,7 +85,7 @@ def get_manifest():
     """
     n = request.args.get('n', type=int, default=10)  # TODO make configurable at app level
     if n > 10:
-        abort(400, 'n too large, max is 10')  # TODO get the max config value here too
+        abort(HTTPStatus.BAD_REQUEST, 'n too large, max is 10')  # TODO get the max config value here too
```

Comment:
is there a wrapper for aborting on 400?

Reply:
No, the flask docs have things like `abort(404)`. It is annoying to be so much wordier with `HTTPStatus.BAD_REQUEST` but I think that's better than a "magic" number even if most everyone knows it.